data/json/effects.json

```diff
@@ -2104,6 +2099,22 @@
     "max_duration": "2 minutes",
     "removes_effects": [ "skunk_spray" ]
   },
+  {
+    "type": "effect_type",
+    "id": "conjunctivitis",
+    "name": [ "Conjunctivitis" ],
+    "desc": [ "Your eyes are inflamed." ],
+    "apply_message": "The chemical irritant damaged your eyes!",
+    "rating": "bad",
+    "resist_effects": [ "took_antihistamine" ],
```

Comment:
antihistamines aren't a treatment for mild chemical burns to your eyes.
```suggestion
```

Reply:
Good call.